### PR TITLE
Move `StepperInput` to make it reusable

### PR DESCRIPTION
**Pull request by robinandeer**
## Describe your changes

- Move out `StepperInput` from price calculator to the global scope

## Justify why they are needed

- Make it reusable, it will be used in QuickAdd component

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* main
  * **PR #3167** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3167" target="_blank">Graphite</a> 
    * **PR #3169** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3169" target="_blank">Graphite</a> 
      * **PR #3170** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3170" target="_blank">Graphite</a> 
        * **PR #3171** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3171" target="_blank">Graphite</a> 
          * **PR #3172** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3172" target="_blank">Graphite</a> 
            * **PR #3177** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3177" target="_blank">Graphite</a>  👈
              * **PR #3178** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3178" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3177?utm_source=stack-comment).

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* main
  * **PR #3167** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3167" target="_blank">Graphite</a> 
    * **PR #3169** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3169" target="_blank">Graphite</a> 
      * **PR #3170** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3170" target="_blank">Graphite</a> 
        * **PR #3171** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3171" target="_blank">Graphite</a> 
          * **PR #3172** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3172" target="_blank">Graphite</a> 
            * **PR #3177** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3177" target="_blank">Graphite</a>  👈
              * **PR #3178** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3178" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3177?utm_source=stack-comment).

File: apps/store/src/components/StepperInput/StepperInput.tsx [hunk 1]
```diff
@@ -25,21 +26,28 @@ export const StepperInput = (props: Props) => {
   const [internalValue, setInternalValue] = useState(value ?? defaultValue ?? min)
   const { highlight, animationProps } = useHighlightAnimation<HTMLDivElement>()
 
-  const increment: MouseEventHandler = (event) => {
+  const changeValue = (event: MouseEvent, amount: 1 | -1) => {
     event.preventDefault()
     highlight()
-    setInternalValue((value) => Math.min(max, value + 1))
+    setInternalValue((value) => {
+      const nextValue = Math.min(max, Math.max(min, value + amount))
```
Comment (alebedev) [inline apps/store/src/components/StepperInput/StepperInput.tsx:33]: We used to check `max` too, let's restore this part maybe?

Comment (robinandeer) [reply apps/store/src/components/StepperInput/StepperInput.tsx:33]: Don't we already do that? 🤔 

But good reminder to add some steps to test this logic

**Review by alebedev — APPROVED** [review]

**Comment by robinandeer** [comment]
### Merge Activity

* **Sep 8, 3:03 PM**: @robinandeer started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3177).

**Pull request merged**